tests/cli/multiple-packages-multiple-worlds.wit.stderr

```diff
@@ -1,3 +1 @@
-error: multiple worlds found; one must be explicitly chosen:
-  test:foo2/some-other-world
-  test:hola/hello
+error: none of the specified packages contain a world
```

Comment:
Could this error be made a bit more descriptive in that it's just the main package which contains no worlds?